roles/download/defaults/main/main.yml

```
@@ -141,21 +141,21 @@ skopeo_version: "v1.13.2"
 kube_major_version: "{{ kube_version | regex_replace('^v([0-9])+\\.([0-9]+)\\.[0-9]+', 'v\\1.\\2') }}"
 
 etcd_supported_versions:
-  v1.27: "v3.5.9"
-  v1.26: "v3.5.9"
-  v1.25: "v3.5.9"
+  v1.28: "v3.5.9"
+  v1.27: "v3.5.7"
+  v1.26: "v3.5.6"
```

```suggestion
  v1.27: "v3.5.9"
  v1.26: "v3.5.9"
```

Creo que lo ideal es dejarlo así, seguramente al resolver conflictos śe quedó ese valor, porque el commit en el que se metió `1.28` sería anterior a la subida a `3.5.9`